scout/server/blueprints/variant/templates/variant/tx_overview.html

```diff
@@ -51,7 +51,18 @@
                       {{ gene.common.hgnc_symbol if gene.common else gene.hgnc_id }}
                     </a>
                   </td> <!-- end of gene symbol col-->
-                  <td>{{ transcript.refseq_identifiers|join(", ") }}</td> <!-- Refseq IDs -->
+                  <td>
+                    {% set decorated_txs = [] %}
+                    {% for tx_id in transcript.refseq_identifiers %}
+                      {% if tx_id.startswith('XM') and transcript.refseq_id != tx_id %}
```

Comment:
👍

Reply:
Without |default("", true) it prints first "none" then the transcript id